youtube_dl/extractor/cbslocal.py

```diff
@@ -60,8 +60,8 @@ class CBSLocalIE(AnvatoIE):
             'title': 'A Very Blue Anniversary',
             'description': 'CBS2’s Cindy Hsu has more.',
             'thumbnail': 're:^https?://.*',
-            'timestamp': 1479962220,
-            'upload_date': '20161124',
+            'timestamp': int,
+            'upload_date': 're:^\d{8}$',
```

Comment:
Ooops, taken it from `sportdeutschland.py`. Now I see that other extractors use `'upload_date': compat_str,`. Which one is better?